src/components/HomePartner/index.tsx

```diff
@@ -5,7 +5,9 @@ export const HomePartner: FC = () => {
   return (
     <SC.HomePartnerWrapper>
       Member of
-      <SC.StyledJetBrainsLogo alt="JetBrains" />
+      <SC.JetBrainsLogoWrapper href="https://jetbrains.com">
```

Comment:
Could you add `rel="noreferrer" target="_blank"`?

https://web.dev/external-anchors-use-rel-noopener/?utm_source=lighthouse&utm_medium=devtools

Reply:
we could also do await with the extra styled component, it's okay to use `<a href="">` directly there